updating/understanding_updates/understanding-openshift-update-duration.adoc

```diff
@@ -58,3 +57,10 @@ include::modules/update-duration-rhel-nodes.adoc[leveloffset=+1]
 
 * xref:../../updating/updating_a_cluster/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating RHEL compute machines]
 endif::openshift-origin[]
+
+[role="_additional-resources"]
+[id="additional-resources_update-duration"]
+== Additional resources
```

Comment:
I moved the prereqs of this page down to the additional resources here, since prereqs for a conceptual page didn't quite make sense to me.

Reply:
they're prereqs because the page content assumes you have already internalized that information before reading the rest of _this_ page.  Possibly worth having someone completely new to OpenShift read this page alone and decide whether they felt lost, or had sufficient local context?